microsoft-edge/dualengine/reference/visiblelistupdateentry.md

```diff
@@ -0,0 +1,72 @@
+---
+description: Represents a change to a travel log entry.
+title: DualEngine Win32 C++ VisibleListUpdateEntry
+author: MSEdgeTeam
+ms.author: edgededev
+ms.prod: microsoft-edge
+ms.date: 11/16/2023
+keywords: dual engine, dualengine, iemode, win32 apps, win32, edge, ie mode, edge html, VisibleListUpdateEntry
+topic_type: 
+- APIRef
+api_name:
+- VisibleListUpdateEntry
+- VisibleListUpdateEntry.fIsFrameEntry
+- VisibleListUpdateEntry.operation
+- VisibleListUpdateEntry.pszTitle
+- VisibleListUpdateEntry.pszUrl
+- VisibleListUpdateEntry.ulEntryId
+api_type:
+- COM
+api_location:
+- ieframe.dll
+---
+
+# struct VisibleListUpdateEntry
+
+> [!IMPORTANT]
+> The DualEngine API is a limited access feature. Contact dualengineapiaccess@microsoft.com for more information.
+
+Represents a change to a travel log entry.
+
+## Summary
+
+ Members                        | Descriptions
+--------------------------------|---------------------------------------------
+[fIsFrameEntry](#fisframeentry) | Indicates whether the entry was for an subframe.

```

Comment:
```suggestion
[fIsFrameEntry](#fisframeentry) | Indicates whether the entry was for a subframe.

```

Reply:
[fIsFrameEntry](#fisframeentry) | Indicates whether the entry was for a subframe.